lib/msal-browser/src/interaction_client/PopupClient.ts

```diff
@@ -114,7 +114,7 @@ export class PopupClient extends StandardInteractionClient {
             const state = this.validateAndExtractStateFromHash(hash, InteractionType.Popup, validRequest.correlationId);
 
             // Remove throttle if it exists
-            ThrottlingUtils.removeThrottle(this.browserStorage, this.config.auth.clientId, authCodeRequest.authority, authCodeRequest.scopes);
+            ThrottlingUtils.removeThrottle(this.browserStorage, this.config.auth.clientId, authCodeRequest, undefined);
```

Comment:
Why `undefined`, why not make this an optional param? Is there a significance?

Reply:
The param is optional, I just missed the fact I could remove the undefined since `homeAccountId` is now the last parameter and is optional. Thanks for catching this!